Apps/UnitTests/Shared/Tests.h

```diff
@@ -98,6 +98,91 @@ TEST(NativeAPI, LifeCycle)
     }
 }
 */
+
+TEST(Performance, Spheres)
+{
+    // create a bunch of sphere, does the rendering for a number of frames, log time it took
+    static const std::string script{ R"(
+        console.log("Setting up Performance test.");
+        var engine = new BABYLON.NativeEngine();
+        var scene = new BABYLON.Scene(engine);
+        function CreateSpheresAsync(scene) {
```

Comment:
Creating spheres does not need to be async. It is only async in the experience to make the code consistent.

Reply:
Fixed